### DISCUSSION TITLE
Updating Docker containers for CNI and Proxy

### DISCUSSION THREAD

**Discussion by pete0emerson** [gruntwork-io/knowledge-base#259] [Help] [answered]
According to AWS support I have to use the ECR `123456789012.dkr.ecr.eu-south-1.amazonaws.com` for cni and proxy.

1 ) How can i update the module? 
2)  Should i except any other issues on Milan region ?  

I am using managed node groups in terraform-aws-eks v0.46.8

**Comment by yorinasub17** [comment, marked as answer]
This is automatically handled by `kubergrunt` in the `sync-core-components` subcommand which the terraform module will call under the hood since version [0.7.7](https://github.com/gruntwork-io/kubergrunt/releases/tag/v0.7.7). So this should resolve itself if you upgrade the version of `kubergrunt`, assuming you have not disabled `kubergrunt` management of the core components (the input variable `use_upgrade_cluster_script`).
